cypress/e2e/user-testing.cy.js

```diff
@@ -0,0 +1,26 @@
+describe('Login', () => {
+  it('logs in with a registered user', () => {
+    cy.visit('/')
+    cy.wait(500)
+    cy.get("#loginEmail").type("mirdahl53192@stud.noroff.no")
+    cy.get("#loginPassword").type("validCredentials")
+    cy.wait(500)
+    cy.get("#loginForm .modal-footer button").contains("Login").click()
+    cy.wait(500)
+    cy.visit('/')
```

Comment:
You are missing a step to "prove" that the login worked. For example checking localStorage for the token.